apps/genetics/src/sections/studyLocus/ColocL2GTable/ColocL2GTable.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import * as d3 from 'd3';
```

Comment:
Something that we need to change is this kind of import for d3. This will import all the helper elements of `d3` into this variable, but in this file, you're only using `d3.ascending`.

Please change the import to `import { ascending } from 'd3';`

This can apply to any other d3 import in the code base.